core/payment-driver/zksync/src/zksync/faucet.rs

```diff
@@ -94,3 +95,12 @@ async fn faucet_donate(address: &str, _network: Network) -> Result<(), GenericEr
     // TODO: Verify tx hash
     Ok(())
 }
+
+async fn resolve_faucet_url() -> Result<String, GenericError> {
+    match env::var(FAUCET_ADDR_ENVAR) {
```

Comment:
Is there any significant cost in `env::var(str)` call. It was previously inside `lazy_static` macro :question:

Reply:
Well, not very significant to be honest. I guess it's fine to leave it here since resolving the missing value requires async context.